pkg/cmd/build.go

```diff
@@ -161,7 +162,25 @@ func doBuild(c *cli.Context, comp *api.Composition) ([]api.BuildOutput, error) {
 	}
 
 	req := &api.BuildRequest{Composition: *comp}
-	resp, err := cl.Build(ctx, req, planDir, sdkDir)
+
+	// if there are extra sources to include for this builder, contextualize
+	// them to the plan's dir.
+	builder := comp.Global.Builder
+	extra, ok := manifest.ExtraSources[builder]
+	if ok && len(extra) != 0 {
```

Comment:
I think this `if` is redundant. You can `range` over a `nil` slice, it is just a no-op.